runtime/src/main/java/io/novafoundation/nova/runtime/storage/source/query/StorageQueryContext.kt

```diff
@@ -103,9 +107,19 @@ interface StorageQueryContext {
     suspend fun <V> Constant.getAs(binding: DynamicInstanceBinder<V>): V
 }
 
-fun Map<StorageEntry, Map<StorageKeyComponents, Any?>>.singleValueOf(storageEntry: StorageEntry) = getValue(storageEntry).values.first()
+@Suppress("DEPRECATION")
+@OptIn(ExperimentalContracts::class)
+suspend fun StorageQueryContext.multi(
+    builderBlock: MultiQueryBuilder.() -> Unit
+): MultiQueryBuilder.Result {
+    contract {
+        callsInPlace(builderBlock, InvocationKind.EXACTLY_ONCE)
+    }
```

Comment:
We cannot define a contract in the interface so this is a workaround to help compiler understand we're invoking `builderBlock` at least once so compiler will understand that assignments present inside `builderBlock` are actually hapenning  to avoid `variable must be itialized` compilation error